josharagon/shufl
# Feature/shuffle animation

## What does this PR do?

- Adds a flip animation on card press
- Adds shuffle count with reset button

## How should it be tested in the UI?

- Npm start the app
- Click on a card, ensure it flips
- click shuffle watch the card move to a random position
- reset the shuffle count to reset the number.
